lib/group_site_manager.py

```diff
@@ -437,12 +439,14 @@ def _upload_oaks(self, numbers) -> None:
       driver.execute_script(f"document.getElementsByTagName('textarea')[0].value = '{js_input}';")
       driver.find_element_by_id('ContentPlaceHolder1_btnGrabar').click()
       time.sleep(2)
+      print("Upload complete for oaks.")
```

Comment:
Why not just print the "finished" notification at the end of `_upload_to_group` ?

Reply:
Just took a look and it looks like the function is returning before we can hit it, for example `_upload_oaks` gets returned from `_upload_to_group` so the finished notification won't get hit.

Reply:
well that's fixable :)

Reply:
Updated to address the returning